openhtf/core/test_record.py

```diff
@@ -72,7 +72,7 @@ class PhaseRecord(  # pylint: disable=no-init
     mutablerecords.Record(
         'PhaseRecord', ['name', 'codeinfo'],
         {'measurements': None,
-         'start_time_millis': None, 'end_time_millis': None,
+         'start_time_millis': int, 'end_time_millis': None,
```

Comment:
What's the motivation to change this from `None` besides being able to cast it in `_asdict()` without a check? It would be really weird, but technically _possible_ for someone to use timestamps that weren't epoch-based, and thus for zero to be a valid timestamp. Effectively defaulting this to zero would create ambiguity in that (admittedly freakish) case.

Aside from that, is the result of calling `int()` part of the definition of Python, or is it implementation specific?